frontend/src/pages/profile.tsx

```
@@ -74,9 +89,11 @@ export default function Profile() {
                         <p>{userData?.email}</p>
                     </div>
                 </div>
-                <button className={styles.deleteButton} onClick={handleDelete}>Delete Account</button>
+                <button className={styles.deleteButton} onClick={handleDelete}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)